search-api/src/main/java/edu/unc/lib/boxc/search/api/SearchFieldKey.java

```diff
@@ -57,6 +57,7 @@
     LANGUAGE("language", "language", "Language"),
     LAST_INDEXED("lastIndexed", "lastIndexed", "lastIndexed"),
     LOCATION("location", "location", "Location"),
+    OTHER_SUBJECTS("otherSubject", "otherSubjects", "Other Subjects"),
```

Comment:
hmm, we're getting a bit inconsistent about pluralization. I'd suggest
```suggestion
    OTHER_SUBJECT("otherSubject", "otherSubject", "Other Subject"),
```
I missed previously that `OTHER_TITLES` was plural, for consistency could you also update that one not to be?